modules/fedimint-mint-client/src/output.rs

```diff
@@ -333,6 +334,13 @@ pub struct NoteIssuanceRequest {
     blinding_key: BlindingKey,
 }
 
+impl hash::Hash for NoteIssuanceRequest {
+    fn hash<H: hash::Hasher>(&self, state: &mut H) {
+        self.spend_key.hash(state);
+        // ignore `blinding_key` as it doesn't impl Hash; `spend_key` has enough
+        // entropy anyway
+    }
+}
```

Comment:
Wondering why clippy isn't complaining that `Hash` is implemented manually while `PartialEq` is being derived. I think technically what you are doing is correct, could just lead to more collisions if someone was to use this data structure in a nonsensical way (same spend key, different blinding keys).